docs/release/release_0_6_0.md

```diff
@@ -55,25 +64,142 @@ number), now is a good time to undo the workarounds for newer versions of
+
+  To restore the orientation from napari 0.5.6 and earlier, change the Depth
+  axis setting to "away" (ie depth axis points away from you).
+
+### Command palette 🎨
+
+Tired of mousing around? Thanks to
+[#5483](https://github.com/napari/napari/pull/5483), napari gains a command
+palette! Press {kbd}`Ctrl/Command+Shift+P` and start typing the name of the
+action you want to use, and press {kbd}`Enter` when you've highlighted it. It
+even works with plugins! This is the culmination of many months of work porting
+napari's actions to Talley Lambert's
+[app-model](https://github.com/pyapp-kit/app-model). 🥳
+
+(video of command palette)
+
+There's still lots of work to be done here, but in the meantime, give it a try!
+We on the team have found it very hard to go back to using napari without the
+palette!
+
+### Feature improvements to Shapes layers
+
+⚠️ *In 0.6.0a1 and earlier, this only works when installing
+PartSegCore-compiled-backend and toggling the "use compiled triangulation"
+option in the advanced preferences. In 0.6.0 it will work with all
+triangulation backends.* ⚠️
+
+Finally, napari Shapes layers can now display polygons with holes in them,
+which starts to open it up for use with mapping data, among other things!
+([#7566](https://github.com/napari/napari/pull/7566),
+[#6654](https://github.com/napari/napari/pull/6654)]) Implementing this feature
+also eliminated a lot of bugs in our polygon drawing code, which could cause
+crashes. If you've had issues with Shapes layers before, now might be a good
+time to give them another try!
+
+### Transition to npe2 plugin engine
+
+npe2 was introduced over four years ago, with napari 0.4.12. npe2 has paved the
+way for new plugin functionality, such as [adding menu
+items](nap-6-contributable-menus) and the command palette. We are now beginning
+the process of deprecating npe1 (napari-plugin-engine) plugins, which we need
+to do to continue to improve npe2 functionality, for example in file readers,
+which is currently very entangled with npe1 code.
+
+To aid this migration, npe1 plugins will now be automatically converted to npe2
+by default. This may break some features if the plugins relied on import-time
+behavior. ([#7627](https://github.com/napari/napari/pull/7627))
+
+During the 0.6.x series, if some plugin functionality is broken by the
+automatic conversion, you can turn off this conversion in the plugin
+preferences. However, the option to turn not convert npe1 plugins will be
+removed in 0.7.0, and npe1 plugins will *only* work through automatic
+conversion. We anticipate 0.7.0 will be released in the second half of 2025.
+
+If you encounter conversion issues in a plugin you rely on, please contact the
+plugin authors to encourage them to migrate their plugin to the npe2 system.
+
+If you are a plugin author and your plugin is not yet npe2-compatible, please
```

Comment:
I would add a link to the more [detailed guide](https://napari.org/dev/plugins/advanced_topics/adapted_plugin_guide.html#adapted-plugin-guide) here e.g. 

```suggestion
For more detail on this change and how it affects plugins, see the [detailed guide](adapted-plugin-guide). If you are a plugin author and your plugin is not yet npe2-compatible, please
```